code/index.html

```diff
@@ -1,22 +1,166 @@
+        <div>
+          <img class="small-news-picture" src="images/bukett_kvadrat.jpg" alt="flowers image">
+          <p>Dried flowers are in style</p>
+          <button>Read More</button>
+      </a>
+      </div>
+    </section>
+
+    <section class="small-news">
+      <a href="https://www.thespruceeats.com/how-to-preserve-berries-1327687" target="_blank" rel="noopener noreferrer">
+        <div>
+          <img class="small-news-picture" src="images/burksommar2 kvadrat.jpg" alt="wild strawberries image">
+          <p>Save the taste of summer</p>
+          <button>Read More</button>
+      </a>
+      </div>
+    </section>
+
+    <section class="small-news">
+      <a href="https://androuet.se/" target="_blank" rel="noopener noreferrer">
+        <div>
+          <img class="small-news-picture" src="images/ost1.jpg" alt="cheese board image">
+          <p>Cheese pairing like in France</p>
+          <button>Read More</button>
+      </a>
+      </div>
+    </section>
+
+    <section class="small-news">
+      <div id="food">
+        <a href="https://www.vivavinomat.se/recept/friterade-torskbollar/" target="_blank" rel="noopener noreferrer">
+          <div>
+            <img class="small-news-picture" src="images/torskbollar_insta.jpg" alt="deep fried cod image">
+            <p>Spanish tapas for the win!</p>
+            <button>Read More</button>
+        </a>
+      </div>
+      </div>
+    </section>
+
+    <section class="small-news">
+      <a href="https://www.vivavinomat.se/recept/restwrap-med-kalvstek-rodbetor/" target="_blank"
+        rel="noopener noreferrer">
+        <div>
+          <img class="small-news-picture" src="images/restwrap_insta.jpg" alt="wrap image">
+          <p>Leftover luxury wrap</p>
+          <button>Read More</button>
+      </a>
+      </div>
+    </section>
+
+    <section class="small-news">
+      <div id="wines">
+        <a href="https://www.systembolaget.se/sok-dryck/?subcategory=R%C3%B6tt%20vin&combinedcountry=Frankrike"
+          target="_blank" rel="noopener noreferrer">
+          <div>
+            <img class="small-news-picture" src="images/hostbukett_vin_insta.jpg" alt="flowers on blue table image">
+            <p>French reds for the weekend</p>
+            <button>Read More</button>
+        </a>
+      </div>
+      </div>
+    </section>
+
+    <section class="small-news">
+      <a href="https://www.vivavinomat.se/recept/pizzabullar-med-pesto-orter/" target="_blank"
+        rel="noopener noreferrer">
+        <div>
+          <img class="small-news-picture" src="images/pizzasnurror_insta2.jpg" alt="pizza buns image">
+          <p>Pizza Pesto Buns recipe</p>
+          <button>Read More</button>
+      </a>
+      </div>
+    </section>
+
+  </main>
+
+  <footer>
+    <div id="footer">
+      <div class="contact">
+        <a href="mailto:tabletop@tabletop.se">Send us an email</a>
+      </div>
+
+      <p class="p-footer">Design by Hanna Rüter</p>
```

Comment:
Do you need this class? Here you could get this element by `footer p {}` in the css